src/app/dynamic-forms/fields/link-multichoice/link-multichoice.component.ts

```diff
@@ -393,7 +392,10 @@ export class LinkMultichoiceComponent implements OnChanges, OnDestroy, OnInit {
             next: (): void => {
 
               this.fileInput.nativeElement.value = null;
-              this._toastr.success("File(s) uploaded", "Success");
+
+              this._notificationService.displaySuccessMessage(
+                `${(this.downloadableLinks.length > 1) ? "Files" : "File"} uploaded`
+              );
```

Comment:
Do we need to account for downloadableLinks to be null?

Reply:
On line 373 it gets set to an empty array if it is unset, so that's already been accounted for